SwiftUIDZ/SwiftUIDZ/VPNView.swift

```diff
@@ -0,0 +1,104 @@
+//
+//  VPNView.swift
+//  SwiftUIDZ
+//
+//  Created by Анна Сычева on 21.01.2023.
+//
+
+import SwiftUI
+
+/// Экран включения VPN
+struct VPNView: View {
+    
+    // MARK: - Constants
+    
+    private enum Constants {
+        static let emptyString = ""
+        static let zeroInt = 0
+        static let twoHundredFloat: CGFloat = 200
+        static let ipText = "IP Address"
+        static let receivedText = "Received"
+        static let sentText = "Sent"
+        static let fiftySpacing: CGFloat = 50
+        static let thirtySpacing: CGFloat = 30
+        static let mbText = "MB"
+        static let serviceText = "Service will expire after 15 days"
+        static let range: ClosedRange<Int> = 0...100
+    }
+    
+    // MARK: - Body
+    
+    var body: some View {
+        VStack(spacing: Constants.twoHundredFloat) {
+            VStack(spacing: Constants.fiftySpacing) {
+                countryPickerView
+                flagImageView
+                vpnToggleView
+                textsView
+            }
+            Text("\(Constants.serviceText)")
+        }
+        .onReceive(timer) { _ in
+            if isOnToggle {
+                receivedCount += Int.random(in: Constants.range)
+            }
+        }
+    }
+    
+    // MARK: - Private Properties
+    
+    private let countries = Country.getCountries()
+    private let timer = Timer
```

Comment:
```suggestion
    private let timer = Timer
```
это должно быть во viewModel